index.html

```diff
@@ -50,6 +48,87 @@ <h2>Software Developer</h2>
           <li>Terminal</li>
         </ul>
       </section>
+      <section class="grid work">
+        <div class="container handles">
+          <img src="media/icons/github.svg" alt="github">
+          <img src="media/icons/linkedin.svg" alt="linkedin">
+          <img src="media/icons/peace.svg" alt="peace">
+          <img src="media/icons/twitter.svg" alt="twitter">
+          <img src="media/icons/m.svg" alt="m">
+        </div>
+        <div>
+          <h4>↓ Work</h4>
+        </div>
+        <div>
+          <h2>MY RECENT <br> WORK</h2>
+        </div>
+      </section>
+      <section class="grid projects">
+        <article class="container project">
+          <img src="https://picsum.photos/327/236?grayscale" alt="placeholder">
+          <h3>Multi-Post Stories</h3>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</p>
+          <div class="container languages">
+            <p>css</p>
+            <p>html</p>
+            <p>bootstrap</p>
+            <p>Ruby</p>
+          </div>
+          <button>
+          See Project
+          </button>
+        </article>
+        <article class="container project">
+          <img src="https://picsum.photos/327/236?grayscale" alt="placeholder">
+          <h3>Multi-Post Stories</h3>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</p>
+          <div class="container languages">
+            <p>css</p>
+            <p>html</p>
+            <p>bootstrap</p>
+            <p>Ruby</p>
+          </div>
+          <button>
+          See Project
+          </button>
+        </article>
+        <article class="container project">
+          <img src="https://picsum.photos/327/236?grayscale" alt="placeholder">
+          <h3>Multi-Post Stories</h3>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, 
+            when an unknown printer took a standard dummy text.</p>
+          <div class="container languages">
+            <p>css</p>
+            <p>html</p>
+            <p>bootstrap</p>
+            <p>Ruby</p>
+          </div>
+          <button>
+          See Project
+          </button>
```

Comment:
- [ ] The button ("See project") is a button tag which you did correctly, should follow the button styling: Active, Hover, and Pressed (see design below)
![image](https://user-images.githubusercontent.com/60772732/125181831-986ba180-e200-11eb-9b9c-321ba6df6c12.png)
Please follow this design for all your buttons as it makes your User Experience even more better.

Reply:
Well resolved